README.markdown

```diff
@@ -245,6 +247,14 @@ See the [documentation](./lib/ngx/re.md) for this Lua module for more details.
 
 [Back to TOC](#table-of-contents)
 
+## ngx.process
+
+This Lua module implements is used to manage the nginx process in Lua.
```

Comment:
This sentence has grammar issues.

"implements is used to" does not make any sense.